config/testing.yml

```diff
@@ -7,7 +7,8 @@ mailer:
   defaultEmailName: 'DEFAULT_EMAIL_NAME'
   forgetPasswordMailTemplateId: 'FORGOT_PASSWORD_MAIL_TEMPLATE_ID'
 
-enableSMS: true
+sms:
```

Comment:
Should be disabled no?

Reply:
Yeah, it should be but It was previously enabled so I didn't modify its value.